collectors/o365/o365_collector.js

```diff
@@ -49,13 +46,21 @@ class O365Collector extends PawsCollector {
             endTs = moment(startTs).add(24, 'hours').toISOString();
```

Comment:
please check `console.log` message above and change it to `console.info/warn` with some message id.